src/__tests__/ParserComponent.test.tsx

```diff
@@ -352,18 +380,33 @@ describe('<ParserComponent /> Unit Tests', () => {
 
     it('handles regular data', () => {
       const t1 = pc.inferTypes(data);
-      // test string
-      expect(t1[2].drawType).toBe(enumDrawType.occurrence);
-      // test number
-      expect(t1[0].drawType).toBe(enumDrawType.any);
-      // test date
-      // expect(t1[1].drawType).toBe(enumDrawType.any);
+      if (t1 !== undefined) {
```

Comment:
Shouldn’t we fail if it’s undefined?

Reply:
If the if statement is not present it highlights t1 and says that it could possibly be undefined